antsibull.cfg

```diff
@@ -10,6 +10,7 @@ pypi_url = https://pypi.org/
 thread_max = 15
 max_retries = 10
 file_check_content = 262144
+# collection_cache =
```

Comment:
This could use a sample value, or some explanation (if you meant to leave it in here commented out). I'm unsure because it doesn't look like there are any other such sample/example comments.

Reply:
I was also unsure, but really like the idea and now added an example: a0c9dcf63b9d8e16744a7b89f339b92bd33dc5a5